cimpl/config.json

```diff
@@ -16,5 +16,35 @@
                 }
             ]
         }
-    ]
+    ],
+    "filterStrategy": {
+        "filter": true,
+        "strategy": "element",
```

Comment:
you could change the filterStrategy to "strategy": "hybrid" which will give you the flexibility to add a combination of both namespaces and individual elements.